examples/LoRaReceiverCallbackWithLNAGain/LoRaReceiverCallbackWithLNAGain.ino

```diff
@@ -0,0 +1,47 @@
+#include <SPI.h>
+#include <LoRa.h>
+
+#ifdef ARDUINO_SAMD_MKRWAN1300
+#error "This example is not compatible with the Arduino MKR WAN 1300 board!"
+#endif
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial);
+
+  Serial.println("LoRa Receiver Callback");
+
+  if (!LoRa.begin(915E6)) {
+    Serial.println("Starting LoRa failed!");
+    while (1);
+  }
+
+  // set maximum LNA for better RX sensitivity
+  // 0: ADC is used and LNA gain is not used.
+  // 1-6: ADC is not used and LNA gain is used.
+  LoRa.setGain(6);

```

Comment:
I'm not sure a full example is needed for this. One alternative would be to add it to one of the existing RX examples commented out.

Thoughts?

Reply:
I will remove my example and add the code into a receive callback example

Reply:
Done.
Added 2 lines only.

https://github.com/sandeepmistry/arduino-LoRa/pull/408/commits/b723b4748da64fb33d7018be3d96bbf92bc527ef